std_msgs/Header header
int8 tower_number
